Resources/Prototypes/Datasets/ion_storm.yml

```
@@ -548,7 +572,7 @@
   - ГОВОРИТЬ ПРАВДУ
   - ВЫКЛЮЧАТЬ СВЕТ
   - ШЕПТАТЬ
-  - BE A REAL SCRUNGLE, JUST A CUTE LIL' GUY
+  # - BE A REAL SCRUNGLE, JUST A CUTE LIL' GUY
```

Ну перевел бы, ей-богу
```suggestion
   - БЫТЬ НАСТОЯЩЕЙ МИЛАШКОЙ, ПРОСТО МИЛЕНЬКИМ СОЗДАНИЕМ
```